### ISSUE TITLE
Audit App Headings in Disability Rating (Benefits)

### ISSUE THREAD

**Issue by brea11y** [department-of-veterans-affairs/va-mobile-app#7346]
Audit the section of the app to ensure that the accessibilityRole=(‘header’) is in place on anything that should be a heading.

- [x] Assign a category to each "heading"
- [x] List location for each "heading"
- [x] List content included in each "heading"
- [x] Listen to each "heading" in VoiceOver or check code to determine if it is currently labeled as a heading
- [x] Determine if changes are needed (if so, include information of what the change should be)
